### PR TITLE
Add notes to the Replay set up and use

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: dev/source/docs/testing-with-replay.rst [hunk 1]
```diff
@@ -8,24 +8,22 @@ Introduction
...
 Replay is a program that takes a dataflash log file and replays it
-through the latest master code allowing for state estimation issues to
-be analysed with new code.
+through any branch's code allowing for state estimation issues to
+be analyzed using that branch's code instead of the code used while generating the dataflash log.
 
-It is recommended that if a problem is reproducible that the log be
-generated with both the `LOG_REPLAY` and `LOG_DISARMED` parameters set
-to 1.
+.. note:: In order to use Replay the log must be generated with both the `LOG_REPLAY` and `LOG_DISARMED` parameters set to 1.
```
Comment (peterbarker) [inline dev/source/docs/testing-with-replay.rst:14]: This is not true; it is just recommended.

Logs do not Replay as well if those parameters are set, but tridge did try to make that work.

File: dev/source/docs/testing-with-replay.rst [hunk 2]
```diff
@@ -8,24 +8,22 @@ Introduction
 ============
 
 Replay is a program that takes a dataflash log file and replays it
-through the latest master code allowing for state estimation issues to
-be analysed with new code.
+through any branch's code allowing for state estimation issues to
+be analyzed using that branch's code instead of the code used while generating the dataflash log.
```
Comment (peterbarker) [inline dev/source/docs/testing-with-replay.rst:12]: A note here saying that the logging structures of the two branches must be the same belongs here....

**Review by peterbarker — CHANGES_REQUESTED** [review]

File: dev/source/docs/testing-with-replay.rst (hunk 1 above)
Comment (Hwurzburg) [reply dev/source/docs/testing-with-replay.rst:14]: if LOG_REPLAY is not enabled, you will not get the replay messages, correct? then what purpose does replaying a log serve?
and what is the impact of not logging prior to arm?
Tridge and Paul had told me I had to have those set to do a proper analysis of an EKF issue...

File: dev/source/docs/testing-with-replay.rst (hunk 2 above)
Comment (Hwurzburg) [reply dev/source/docs/testing-with-replay.rst:12]: will add

**Comment by peterbarker** [comment]
On Fri, 4 Dec 2020, Henry Wurzburg wrote:

> -generated with both the `LOG_REPLAY` and `LOG_DISARMED` parameters set
> -to 1.
> +.. note:: In order to use Replay the log must be generated with both the `LOG_REPLAY` and `LOG_DISARMED` parameters set to 1.
> 
> if LOG_REPLAY is not enabled, you will not get the replay messages, correct? then what purpose does replaying a log serve?

Sorry.  LOG_REPLAY must be set LOG_DISARMED does not - we just get a much, 
much better log if LOG_DISARMED is set.

File: dev/source/docs/testing-with-replay.rst (hunk 1 above)
Comment (Hwurzburg) [reply dev/source/docs/testing-with-replay.rst]: changed to require replay, preferred to also have disarmed set also

**Comment by Hwurzburg** [comment]
should be good to go now..

**Review by peterbarker — APPROVED** [review]
Thanks Henry.

**Pull request merged**